### ISSUE TITLE
Any roadmap on this module? How can I help?

### ISSUE THREAD

**Issue by Oza94** [pixijs/pixi-lights#9]
Hey @englercj !

I really appreciate your work on Pixi and pixi-lights! I would like to know if you have any agenda to push this module on NPM? Also how can I help to speed the development considering I have only a basic knowledge of WebGL/OpenGL (I can understand your code but will never be able to do it myself, especially if we talk about optimization x) )?

I will start to use pixi-lights in my project (with a development flag in the meantime before the first release) and make feedbacks if needed, for the moment this is the best I can do :) don't hesitate if you need some support with docs generation, build process, showcase site or stuff like that.

**Comment by englercj** [comment]
https://github.com/pixijs/pixi-lights#roadmap

I plan to push it to npm when it is ready for use in real projects, right now it is pretty hacky and unoptimized in a few places.

**Comment by harangue** [comment]
Is an eventual merge with Pixi a possibility?

**Comment by englercj** [comment]
Not sure what you mean, the whole point of v3 of pixi was so that we could move most of the functionality out to plugins. The motto is if it doesn't *have* to be in core, it isn't.

**Comment by harangue** [comment]
I haven't used Pixi since v2 so that was news to me. :) Keep up the good work.

**Comment by JTLunsford** [comment]
We would like to use this in a project. Would you be open to pull requests for pixi-lights?

**Comment by englercj** [comment]
Of course, keep in mind that this was an experiment. And that it was never really meant to go beyond that until we are using WebGL2 and have MRTs.

**Comment by paralin** [comment]
@englercj Any updates on this?

**Comment by ivanpopelyshev** [comment]
@paralin 

I'm working on something like that right now

**Comment by paralin** [comment]
@ivanpopelyshev Cool, I'm also curious about https://github.com/oniproject/pixiLights - seems like the Sweep they are doing there is really good and could be integrated. Can I help at all on this?

**Comment by englercj** [comment]
> @englercj Any updates on this?

Not sure what you are looking for, I mentioned a few times this is just an experiment. What is it you are waiting for?

**Comment by ivanpopelyshev** [comment]
@paralin The main question is, why do you need of that? If you want to use MULTIPLY or OVERLAY modes with some pre-defined lights , like that one, then I can tell you how to do that in vanilla pixi-v4

![test](https://pp.vk.me/c631126/v631126922/619f0/LUKxWBhBBBE.jpg)

**Comment by paralin** [comment]
It's a particular visual style where vision is provided by line of sight from the light / emitter. Aka, light is blocked by objects properly, rather than just overlaying everything
